test/functional/kernel/basic-fission-fusion-loop/test-kernel-basic-fission-fusion-loop.cpp.in

```diff
@@ -0,0 +1,175 @@
+      RAJA::statement::Lambda<0, RAJA::Segs<0>>
+    >,
+    RAJA::statement::For<0, RAJA::omp_parallel_for_exec,
+      RAJA::statement::Lambda<1, RAJA::Segs<0>>
+    >
+  >
+
+>;
+#endif
+
+#if defined(RAJA_ENABLE_TBB)
+using TBBKernelExecPols = camp::list<
+
+  RAJA::KernelPolicy<
+    RAJA::statement::For<0, RAJA::tbb_for_exec,
+      RAJA::statement::Lambda<0, RAJA::Segs<0>>,
+      RAJA::statement::Lambda<1, RAJA::Segs<0>>
+    >
+  >,
+
+  RAJA::KernelPolicy<
+    RAJA::statement::For<0, RAJA::tbb_for_exec,
+      RAJA::statement::Lambda<0, RAJA::Segs<0>>
+    >,
+    RAJA::statement::For<0, RAJA::tbb_for_exec,
+      RAJA::statement::Lambda<1, RAJA::Segs<0>>
+    >
+  >
+
+>;
+#endif
+
+#if defined(RAJA_ENABLE_TARGET_OPENMP)
+using OpenMPTargetKernelExecPols = camp::list<
+
+  RAJA::KernelPolicy<
+    RAJA::statement::For<0, RAJA::omp_target_parallel_for_exec<4>,
+      RAJA::statement::Lambda<0, RAJA::Segs<0>>,
+      RAJA::statement::Lambda<1, RAJA::Segs<0>>
+    >
+  >,
+
+  RAJA::KernelPolicy<
+    RAJA::statement::For<0, RAJA::omp_target_parallel_for_exec<4>,
+      RAJA::statement::Lambda<0, RAJA::Segs<0>>>,
+    RAJA::statement::For<0, RAJA::omp_target_parallel_for_exec<4>,
+      RAJA::statement::Lambda<1, RAJA::Segs<0>>
+    >
+  >
+
+>;
+#endif
+
+#if defined(RAJA_ENABLE_CUDA)
+using CudaKernelExecPols = camp::list<
+
+  RAJA::KernelPolicy<
+    RAJA::statement::CudaKernel<
+      RAJA::statement::For<0, RAJA::cuda_thread_x_loop,
+        RAJA::statement::Lambda<0, RAJA::Segs<0>>,
+        RAJA::statement::Lambda<1, RAJA::Segs<0>>
+      >
+    >
+  >,
+
+  RAJA::KernelPolicy<
+    RAJA::statement::CudaKernel<
+      RAJA::statement::For<0, RAJA::cuda_thread_x_loop,
+        RAJA::statement::Lambda<0, RAJA::Segs<0>>
+      >,
+      RAJA::statement::For<0, RAJA::cuda_thread_x_loop,
+        RAJA::statement::Lambda<1, RAJA::Segs<0>>
+      >
+    >
+  >
+
+>;
+#endif
+
+#if defined(RAJA_ENABLE_HIP)
+using HipKernelExecPols = camp::list<
+
+  RAJA::KernelPolicy<
+    RAJA::statement::HipKernel<
+      RAJA::statement::For<0, RAJA::hip_thread_x_loop,
+        RAJA::statement::Lambda<0, RAJA::Segs<0>>,
+        RAJA::statement::Lambda<1, RAJA::Segs<0>>
+      >
+    >
+  >,
+
+  RAJA::KernelPolicy<
+    RAJA::statement::HipKernel<
+      RAJA::statement::For<0, RAJA::hip_thread_x_loop,
+        RAJA::statement::Lambda<0, RAJA::Segs<0>>
+      >,
+      RAJA::statement::For<0, RAJA::hip_thread_x_loop,
+        RAJA::statement::Lambda<1, RAJA::Segs<0>>
+      >
+    >
+  >
+
+>;
+#endif
+
+
+//
+// Cartesian product of types used in parameterized tests
+//
+using @BACKEND@KernelBasicFisionFussionLoopTypes =
+    Test<camp::cartesian_product<StrongIdxTypeList,
+                                 @BACKEND@ResourceList,
+                                 @BACKEND@KernelExecPols>>::Types;
+
+//
+// Instantiate parameterized tests
+//
+INSTANTIATE_TYPED_TEST_SUITE_P(@BACKEND@,
+                               KernelBasicFisionFussionLoopTest,
```

Comment:
Misspelling of `FisionFussion` here and throughout. I'll fix these.

Reply:
Thanks. I missed that. Sorry.